app/webpacker/components/RegistrationsV2/RegistrationAdministration/RegistrationActions.jsx

```diff
@@ -124,7 +127,21 @@ export default function RegistrationActions({
 
   const attemptToApprove = () => {
     const idsToAccept = [...pending, ...cancelled, ...waiting, ...rejected];
-    if (idsToAccept.length > spotsRemaining) {
+    const skippedWaitlistCount = getSkippedWaitlistCount(
+      registrations,
+      partitionedSelected,
+    );
+
+    if (skippedWaitlistCount > 0) {
+      confirm({
+        content: I18n.t(
+          'competitions.registration_v2.list.waitlist.skipped_warning',
+          { count: skippedWaitlistCount },
+        ),
+      }).then(
+        () => changeStatus(idsToAccept, 'accepted'),
+      ).catch(() => null);
```

Comment:
I get an error without the catch, here and in every other use of `useConfirm` in the code, but it doesn't happen on the actual website.

Reply:
Yeah maybe I should look into updating the actual `ConfirmProvider` to fix the underlying issue. Must have something to do with `StrictMode` being enabled/disabled.